packages/blade/src/components/Indicator/__tests__/Indicator.web.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { Indicator } from '..';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+import assertAccessible from '~src/_helpers/testing/assertAccessible.web';
+
+describe('<Indicator />', () => {
+  it('should render', () => {
+    const { container } = renderWithTheme(<Indicator accessibilityLabel="Success" />);
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render different variants for size and intent', () => {
+    const { container } = renderWithTheme(
+      <Indicator accessibilityLabel="Warning" size="small" intent="notice" />,
+    );
+
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should render text label', () => {
+    const label = 'Active';
+    const { getByText } = renderWithTheme(<Indicator>{label}</Indicator>);
+
+    getByText(label);
+  });
+
+  it('should have role and a11y label when text label is passed', async () => {
+    const label = 'Active';
+    const { getByRole, getByLabelText } = renderWithTheme(<Indicator>{label}</Indicator>);
+
+    getByRole('status');
```

Comment:
if we are doing `assertAccessible` then do we need this `getByRole`? doesn't `assertAccessible` checks for role as well?

Reply:
Here in `getByRole` we assert that the role should be `status`. `assertAccessible` will highlight issues where we can forget passing a role but it doesn't know that the `role` should be `status`.